docs/make.jl

```diff
@@ -12,7 +12,6 @@ makedocs(
     format = DocumenterVitepress.MarkdownVitepress(
         repo = "https://github.com/BioJulia/BioJuliaDocs",
         md_output_path = ".",
-        build_vitepress = false
```

Comment:
this could also be `haskey(ENV, "CI")` if you want

Reply:
ahhh, missed that. cool, thanks!